logs-lib/logs/panels.libsonnet

```diff
@@ -17,12 +18,19 @@ function(
     logsVolumeInit(targets, title='Logs volume')::
       timeSeries.new(title)
       + timeSeries.queryOptions.withTargets(targets)
+      + timeSeries.panelOptions.withDescription('Logs volume grouped by "%s" label.' % logsVolumeGroupBy)
+      // set type to first target's type
+      + timeSeries.queryOptions.withDatasource(
+        logsVolumeTarget.datasource.type, logsVolumeTarget.datasource.uid
+      )
       + custom.withDrawStyle('bars')
       + custom.stacking.withMode('normal')
       + custom.withFillOpacity(50)
-      + timeSeries.queryOptions.withInterval('30s')  // must be set , otherwise interval is around 1ms
+      // should be set, otherwise interval is around 1s by default
+      + timeSeries.queryOptions.withInterval('30s')
```

Comment:
Should this be a parameter somewhere? Or, perhaps in the common-lib we add a generic "panels utility" function that can update this on all panels you pass it.

Given how this "broke" so many dashboards for me, and how crucial it can be in getting the right display for a given query, it shouldn't be as hidden.

I do agree with setting a sane default value tho.

Reply:
Indeed.  I'll try to address this hardcode next time I change something in this lib.

Reply:
I think an approach similar to Shahzeb's for the status panels, with how they ingest the integration version from the metadata might make sense? 

I wouldn't be against setting the default update interval in either the metadata or config, but we'd have to be consistent in using it